test/unittests/t_catalog_mgr.cc

```diff
@@ -20,7 +20,6 @@ using namespace std;  // NOLINT
 namespace catalog {
 
 class T_CatalogManager : public ::testing::Test {
-
```

Comment:
All the changes in this file, do they belong to the new feature or are they fixes?